contracts/evoting/evoting.go

```diff
@@ -643,17 +643,20 @@ func (e evotingCommand) combineShares(snap store.Snapshot, step execution.Step)
 		return xerrors.Errorf("failed to create dir: %v", err)
 	}
 
-	tmp, err := ioutil.TempDir(e.exportFolder, "decrypted-ballots")
-	if err != nil {
-		return xerrors.Errorf("failed to create temp dir: %v", err)
-	}
+	// we must pass to the Unikernel the RELATIVE folder path from the one
+	// mounted, i.e. e.exportFolder. This folder MUST be uniq for each
+	// transaction.
+	txIDHex := hex.EncodeToString(step.Current.GetID())
+	tmp := fmt.Sprintf("decrypted-ballots-%s-%s-%d/", tx.ElectionID[:4], txIDHex[:4], time.Now().UnixNano())
 
-	err = exportPubshares(tmp, election.PubsharesUnits, numBallots, numChunks)
+	dela.Logger.Info().Msgf("using the following folder: %s", tmp)
+
+	err = exportPubshares(filepath.Join(e.exportFolder, tmp), election.PubsharesUnits, numBallots, numChunks)
```

Comment:
`filepath.Join(e.exportFolder, tmp)` is recomputed multiple times.
Consider using a variable.